flask_multipass/core.py

```diff
@@ -134,10 +134,13 @@ def validate_next_url(self, url):
         If you override this and want to allow more hosts, make sure to use
         a whitelist of trusted hosts to avoid creating an open redirector.
         """
-        url_info = urlsplit(url)
+        # Browsers treat backslashes like forward slashes, while urllib doesn't.
+        # Since we just want to validate scheme and netloc here, we normalize
+        # slashes to those recognized by urllib.
+        url_info = urlsplit(url.replace('\\', '/'))
```

Comment:
I'm not sure if there are cases where the backslash has a function like in older/unusual browsers (e.g. some Japanese institutions still use Internet Explorer :( ).
So I didn't use replace() but yes it seems logical.

Reply:
ahah wise decision :+1: